MachineLearning_Engine/MachineLearning_Engine.csproj

```diff
@@ -48,6 +48,10 @@
       <HintPath>..\..\Python_Toolkit\Build\Python_Engine.dll</HintPath>
       <Private>False</Private>
     </Reference>
+    <Reference Include="Reflection_oM, Version=3.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\BHoM\Build\Reflection_oM.dll</HintPath>
+    </Reference>
```

Comment:
Reference should not have version, culture, or processorArchitecture.
Should also be set to copy local false :smile: